test-framework/core/src/main/java/org/keycloak/testframework/https/CertificatesConfigBuilder.java

```diff
@@ -0,0 +1,60 @@
+package org.keycloak.testframework.https;
+
+import org.keycloak.common.util.KeystoreUtil;
+
+public class CertificatesConfigBuilder {
+
+    private CryptoProviderOption cryptoProviderOption = CryptoProviderOption.DEFAULT;
+    private KeystoreUtil.KeystoreFormat keystoreFormat = KeystoreUtil.KeystoreFormat.JKS;
+    private String fileFormatSuffix = ".jks";
```

Comment:
I wouldn't deal with the file suffix here (it's not something we need to make configurable); just handle that in the supplier.